### ISSUE TITLE
Discussion email list or forum 

### ISSUE THREAD

**Issue by jasonbyun8** [renode/renode#428]
I have been enjoying using renode so far, but if I come across to any questions, I cannot find any good place to search or ask except the doc from renode website.

QEMU comes with discussion mail list --  https://www.qemu.org/support/ -- which really helps to learn material quickly and enables user to help each other. 

It would be really great if renode also comes with such features; email list, forum, discord, telegram, etc.

**Comment by sulemankm** [comment]
Yeah me too!  Renode seems to be a very good piece of software.  However, a public discussion/chat forum is direly needed and would be great to have!

**Comment by otavio** [comment]
Agreed. This would help in the tool adoption.
